build.gradle

```diff
@@ -20,9 +24,9 @@ android {
     }
 
     // To let the libs be included in other projects
-    sourceSets {
```

Comment:
Unfortunately, it seems that, unlike for Android platforms, there is no common standard on

1. how to cross compile JNI libraries
2. how to package cross compiled JNI libraries (and how to name the platforms such that compatible platforms have the same name and incompatible platforms have a different name)
3. how to load cross compiled JNI libraries
4. how to test cross compiled JNI libraries for all supported target platforms

on these platforms:

1. Linux x86,
2. Linux x86-64,
3. Linux armv6,
4. Linux armv7,
5. Linux armv8-a,
6. Windows x86,
7. Windows x86-64, 
8. MacOS universal binary

At least I’m not aware of such a standard. It also seems that creating and implementing such a standard is substantial work, as I can imagine that

1. on certain platforms, libraries need to be unpacked (from .jar files) before they can be loaded
2. the probability of cross compiler toolchains being available to the average developer for all target platforms is low, leading to build failures if we attempt to implement this, unless we first download and compile the cross compiler toolchains for all target platforms...

That’s why I’d consider this a separate issue.